docs/config.md

```diff
@@ -653,6 +653,18 @@ perf_analyzer_flags:
     latency-report-file: /path/to/latency/report/file
 ```
 
+The `input-data`, `shape`, and `streaming` perf_analyzer options are additive and can take either a single string (non-list) or a list of strings:
+
+```yaml
+...
+perf_analyzer_flags:
+  input-data: my-input-data-1
+  shape:
+    - my-shape-1
```

Comment:
Could you please change this to a valid shape argument for perf_analyzer_flags? For example, INPUT0:1024,1024 instead of my-shape-1.

Reply:
changed 👍